tests/Http/Place/UpdateAddressRequestHandlerTest.php

```diff
@@ -0,0 +1,255 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Place;
+
+use Broadway\CommandHandling\Testing\TraceableCommandBus;
+use CultuurNet\UDB3\Address\Address;
+use CultuurNet\UDB3\Address\Locality;
+use CultuurNet\UDB3\Address\PostalCode;
+use CultuurNet\UDB3\Address\Street;
+use CultuurNet\UDB3\Http\ApiProblem\AssertApiProblemTrait;
+use CultuurNet\UDB3\Http\Request\Psr7RequestBuilder;
+use CultuurNet\UDB3\Language;
+use CultuurNet\UDB3\Model\ValueObject\Geography\CountryCode;
+use CultuurNet\UDB3\Place\Commands\UpdateAddress;
+use CultuurNet\UDB3\ReadModel\InMemoryDocumentRepository;
+use CultuurNet\UDB3\ReadModel\JsonDocument;
+use InvalidArgumentException;
+use PHPUnit\Framework\TestCase;
+
+class UpdateAddressRequestHandlerTest extends TestCase
+{
+    use AssertApiProblemTrait;
+
+    private const PLACE_ID = 'd2a039e9-f4d6-4080-ae33-a106b5d3d47b';
+
+    private TraceableCommandBus $commandBus;
+
+    private UpdateAddressRequestHandler $updateAddressRequestHandler;
+
+    private Psr7RequestBuilder $psr7RequestBuilder;
+
+    protected function setUp(): void
+    {
+        $this->commandBus = new TraceableCommandBus();
+
+        $placeDocumentRepository = new InMemoryDocumentRepository();
+
+        $placeDocumentRepository->save(
+            new JsonDocument(self::PLACE_ID, '{}')
+        );
+
+        $this->updateAddressRequestHandler = new UpdateAddressRequestHandler(
+            $this->commandBus,
+            $placeDocumentRepository
+        );
+
+        $this->psr7RequestBuilder = new Psr7RequestBuilder();
+
+        $this->commandBus->record();
+    }
+
+    /**
+     * @test
+     */
+    public function it_handles_updating_the_address(): void
+    {
+        $updateAddressRequest = $this->psr7RequestBuilder
+            ->withRouteParameter('placeId', self::PLACE_ID)
+            ->withRouteParameter('language', 'nl')
+            ->withBodyFromString(
+                '{
+                    "streetAddress": "Veldstraat 11",
+                    "postalCode": "9000",
+                    "addressLocality": "Gent",
+                    "addressCountry": "BE"
+                }'
+            )
+            ->build('PUT');
+
+        $this->updateAddressRequestHandler->handle($updateAddressRequest);
```

Comment:
Although the response is `NoContentResponse` I would still check it with `AssertJsonResponseTrait`